### PR TITLE
wire-desktop: update to 3.24.2939

### PR DESCRIPTION
**Pull request by Veganizer**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86-64)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

### CODE REVIEW

**Comment by Veganizer** [comment]
@yjp20 could you check please?

**Comment by yjp20** [comment]
lgtm, but it's been a while since I actively maintained packages for voidlinux.

**Comment by Veganizer** [comment]
@yjp20 I just wanted to make sure that you approve my changes. If you want I will maintain it. But would be nice if you want to continue the maintenance :-)

File: srcpkgs/wire-desktop/files/wire-desktop.desktop [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Exec=wire-desktop %U
+Terminal=false
+Type=Application
+Icon=wire-desktop
+StartupWMClass=Wire
+Categories=Network;
+GenericName=Secure messenger
+Keywords=chat;encrypt;e2e;messenger;videocall
+MimeType=x-scheme-handler/wire
+Version=1.1
```
Comment (ericonr) [inline srcpkgs/wire-desktop/files/wire-desktop.desktop:13]: It's an open source project; please have this upstream instead of vendoring it in.

Comment (Veganizer) [reply srcpkgs/wire-desktop/files/wire-desktop.desktop:13]: I don't understand what you mean? Wire isn't creating a desktop file by itself.

Comment (ericonr) [reply srcpkgs/wire-desktop/files/wire-desktop.desktop:13]: Yes, but it's their responsibility. Make a PR to them, don't create this file here.

Comment (Veganizer) [reply srcpkgs/wire-desktop/files/wire-desktop.desktop:13]: https://github.com/wireapp/wire-desktop/issues/4920
Wrote a issue there now.

**Pull request merged**